pytorch/training/docker/2.0/py3/cu121/Dockerfile.ec2.gpu.os_scan_allowlist.json

```diff
@@ -23,6 +23,32 @@
       "severity": "HIGH",
       "status": "ACTIVE",
       "title": "CVE-2020-11979 - org.apache.ant:ant"
+    },
+  ],
+  "torch": [
+    {
+      "description": "Overview\n[torch](https://pypi.org/project/torch) is a Tensors and Dynamic neural networks in Python with strong GPU acceleration\n\nAffected versions of this package are vulnerable to Arbitrary Code Injection through `filter-test-configs` in the `pull_request_target-triggered` workflow, allowing an attacker to use a malicious branch name to gain command execution and potentially leak secrets.",
```

Comment:
There seems to be a mismatch in the description reported by the scanner and the one provided in this JSON.